metricflow/api/metricflow_client.py

```diff
@@ -247,13 +235,12 @@ def drop_materialization(self, materialization_name: str) -> bool:
         Returns:
             True if a table has been drop, False if table doesn't exist.
         """
-        return self.mf_engine.drop_materialization(materialization_name=materialization_name)
+        return self.engine.drop_materialization(materialization_name=materialization_name)
 
-    def validate_configs(self) -> List[ValidationIssueType]:
+    def validate_configs(self) -> Optional[Tuple[ValidationIssueType, ...]]:
```

Comment:
Ok. I'm going to open an issue for this - I don't think validators should return Optional[Tuple[...]] - and then this callsite should get cleaned up along with the rest.

Reply:
Filed #49